react-components/src/utils/test-utils.js

```diff
@@ -14,14 +14,41 @@
 
 import React from 'react';
 import { render } from '@testing-library/react';
-import { MockedProvider } from '@apollo/react-testing';
+import { MockedProvider } from '@apollo/client/testing';
 
 import QUERY_CART_DETAILS from '../queries/query_cart_details.graphql';
+import QUERY_COUNTRIES from '../queries/query_countries.graphql';
+import QUERY_CUSTOMER_CART from '../queries/query_customer_cart.graphql';
+import MUTATION_PLACE_ORDER from '../queries/mutation_place_order.graphql';
+import MUTATION_SET_SHIPPING_ADDRESS from '../queries/mutation_set_shipping_address.graphql';
 
 const emptyCartId = 'empty';
 const mockCartId = '123ABC';
+const mockShippingAddress = {
```

Comment:
I think we should improve this mechanism a bit, like have these mocks "collected" at runtime from a file structure instead of just piling them into this file.